From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Sat, 9 Apr 2022 21:54:30 +0100
Subject: manpage subdir

---
 src/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/meson.build b/src/meson.build
index 3b0fc24..acce6aa 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -74,7 +74,7 @@ if pod2man.found()
                                      '--center="Transmission Remote GTK"',
                                      '@INPUT@', '@OUTPUT@'],
                            install: true,
-                           install_dir: get_option('mandir'))
+                           install_dir: get_option('mandir') / 'man1')
 endif
 
 # gresources.xml
