From: Benjamin Barenblat <bbaren@debian.org>
Date: Tue, 5 Nov 2024 13:49:35 +0000
Subject: Prevent tests from attempting network access

Forwarded: not-needed
---
 data/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/meson.build b/data/meson.build
index 68412a7..3c36eba 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -37,6 +37,6 @@ appdata_file = i18n.merge_file(input: '@0@.appdata.xml.in'.format(app_id),
 
 appstream_util = find_program('appstream-util', required: false)
 if appstream_util.found()
-  test('Validate appdata.xml', appstream_util, args: ['validate-relax', appdata_file])
+  test('Validate appdata.xml', appstream_util, args: ['--nonet', 'validate-relax', appdata_file])
 endif
 
